Assets/Scripts/Player/Level generation scripts/TavernGenerationScript.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+
+public class TavernGenerationScript : TileGenerationScript
+{
+    public override GameObject Generate(int column, int row)
+    {
+        //we have the tavern tiles.
+        if (column % (Size - 1) == 1)
+        {
+            return Instantiate(tiles[0], new Vector3(column * Width, 0, row * Width), 
```

Comment:
I don't understand why `tiles` is a list, because you are always using `tiles[0]`. Shouldn't you be using a random value instead of 0 all the time ?

Reply:
Tiles is a list for when we start to make multiple blocks with different obstacles on them. For now i made it get the first one since this was just to make the level generation. The next feature will work on making more blocks